pom.xml

```diff
@@ -90,10 +90,16 @@
         <hdf5.version>1.10.5</hdf5.version>
         <hdf5.javacpp.version>${hdf5.version}-${javacpp.version}</hdf5.javacpp.version>
         <cuda>10.1</cuda>
+        <reactive-streams.version>1.0.3</reactive-streams.version>
     </properties>
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>org.reactivestreams</groupId>
+                <artifactId>reactive-streams</artifactId>
```

Comment:
What's reactive streams for?

Reply:
This was to solve the following enforcing rule issue: 
```
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-maven) @ konduit-serving-war ---
[WARNING] 
Dependency convergence error for org.reactivestreams:reactive-streams:1.0.2 paths to dependency are:
+-ai.konduit.serving:konduit-serving-war:1.2.1
  +-io.vertx:vertx-rx-java2:3.8.3
    +-io.reactivex.rxjava2:rxjava:2.2.12
      +-org.reactivestreams:reactive-streams:1.0.2
and
+-ai.konduit.serving:konduit-serving-war:1.2.1
  +-io.vertx:vertx-rx-java2:3.8.3
    +-org.reactivestreams:reactive-streams:1.0.3
[WARNING] Rule 1: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:
Rule 1: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:


Failed while enforcing releasability. See above detailed error message.
```

Reply:
@ShamsUlAzeem  can you put that under dependency management then? That's what it's for. It doesn't need to be a direct dependency. It's a transitive dependency already. That will force the proper version to be pulled in.

Reply:
@agibsonccc it's already under dependency management.